biz.ganttproject.core/src/main/java/biz/ganttproject/core/chart/scene/gantt/TaskTableSceneBuilder.kt

```diff
@@ -0,0 +1,59 @@
+/*
+Copyright 2021 BarD Software s.r.o, Dmitry Kazakov
+
+This file is part of GanttProject, an open-source project management tool.
+
+GanttProject is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+GanttProject is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with GanttProject.  If not, see <http://www.gnu.org/licenses/>.
+*/
+package biz.ganttproject.core.chart.scene.gantt
+
+import biz.ganttproject.core.chart.scene.gantt.TableSceneBuilder.*
+import biz.ganttproject.core.chart.canvas.Canvas
+import biz.ganttproject.core.chart.canvas.TextMetrics
+
+class TaskTableSceneBuilder(
+  private val input: InputApi
+) {
+  private val tableSceneBuilder = TableSceneBuilder(object : TableSceneBuilder.InputApi {
```

Comment:
Is it possible to convert `TaskTableSceneBuilder.InputApi` to simple data class?